packages/common/src/core/schemas/shared/subschemas.ts

```diff
@@ -53,6 +53,22 @@ export const ENTITY_LOCATION_SCHEMA = {
   default: { type: "none" },
 };
 
+/**
+ * defines the JSON schema for the map configuration settings
+ */
+export const ENTITY_MAP_SCHEMA = {
+  type: "object",
+  properties: {
+    itemId: {
+      type: "array",
```

Comment:
Out of curiosity, why are we making this an array?

Reply:
Mainly just for possible future use cases, such as a map switcher where we would store multiple items.  As current, we just defined maxItems and treat it like a single item id.